dev-libs/libvterm/libvterm-0.1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit flag-o-matic
+
+DESCRIPTION="An abstract library implementation of a VT220/xterm/ECMA-48 terminal emulator"
+HOMEPAGE="http://www.leonerd.org.uk/code/libvterm/"
+SRC_URI="https://launchpad.net/${PN}/trunk/v${PV}/+download/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	dev-lang/perl
```

Comment:
Since you've mentioned this in context of tests, just to be clear: perl is needed unconditionally, or for tests only?

Reply:
Unconditionally, they use it in the normal build procedure too.